src/dsymbol/conversion/first.d

```diff
@@ -848,7 +856,7 @@ private:
 	}
 
 	void pushSymbol(string name, CompletionKind kind, istring symbolFile,
-		size_t location = 0, const Type type = null)
+		size_t location = 0, const Type type = null, const IdType protection = tok!"public")
```

Comment:
this protection argument isn't used

also ditto from previous comment

Reply:
Will fix.